includes/polling/os/unix.inc.php

```diff
@@ -1,5 +1,9 @@
 <?php
 
+echo "---------------------";
+var_export($device);
+echo "---------------------";
```

Comment:
Yes, it can be removed. I was debugging 'cause it was dropping this error.

> Warning: Invalid argument supplied for foreach() in /opt/librenms/includes/polling/unix-agent/packages.inc.php on line 71                                                                                                                                                                 
                                                                                                                                                                                                                                                                                          
> Warning: Invalid argument supplied for foreach() in /opt/librenms/includes/polling/unix-agent/packages.inc.php on line 146

This error is generated 'cause the variables aren't initialized, so I kept some debug code in case there was an error of any kind when fixing it.

Reply:
so please do so ;-)

Reply:
Removed echos.